packages/monitor-v2/src/monitor-og/oSnapAutomation.ts

```diff
@@ -162,8 +170,21 @@ const getSupportedSnapshotProposals = async (
     )
   ).flat();
 
+  // Log all errors that occurred when fetching Snapshot proposals and filter them out.
```

Comment:
Might be a dumb question, but why do you log an error here and on line 406? Would this generate 2 error logs in some cases?

Reply:
This section logs errors when fetching off-chain proposal before posting it on-chain (proposer logic)
Logging in line 406 deals with errors when verifying already submitted proposal (disputer logic)